### ISSUE TITLE
Notification stopped working after Fx v111.0 update

### ISSUE THREAD

**Issue by Parsnip** [Mika-/torrent-control#146]
Nothing much else to add here what isn't on the subject. Fx updated to versino 111 yesterday and today I noticed that torrent control doesn't show the notification anymore when I add a torrent. The torrent gets added just fine, just the notification is missing.

From the Fx changelog, "Windows native notifications are now enabled", which is clearly the change that's affecting this. No surprise then that disabling that feature through about:config brings the build-in notification also back. So while it's possible to disable the Windows native notifications for now, it might not be possible in the future so I guess this is more of a heads up to you in case that happens. I don't know if there are other ways to do notifications on Fx though.

**Comment by Parsnip** [comment]
I forgot to mention, if anyone else wants to turns the new native notifications off,  the about:config string to toggle is 'alerts.useSystemBackend', put that to 'false' to get the old notifications back, at least for now.

**Comment by Mika-** [comment]
Native notifications are working out of the box on my system. Check that Firefox is allowed to show notifications in windows notifications management and notifications are not silenced.

**Comment by Parsnip** [comment]
Yes, they work, but I imagine some people will have the system level notifications disabled like I did and would like to keep them disabled for one reason or another.

**Comment by Mika-** [comment]
Web api only exposes very base level interface for notifications, so you need to continue using flags in about:config to do that. I would expect Firefox to continue supporting internal notifications, as many linux environments don't support them in system level.

**Comment by Parsnip** [comment]
That's a good point about the Linux stuff, was wondering about that myself but have practically zero familiarity with any of the distros so wasn't sure how they would handle that. So that could mean that the internal notifications will stick around, just behind a flag on Windows, for the foreseeable future.

**Comment by DevXen** [comment]
> Native notifications are working out of the box on my system. Check that Firefox is allowed to show notifications in windows notifications management and notifications are not silenced.

So I came here as my notifications are also missing. as per your comment above i checked the notification and actions in my windows. and Firefox is allowed for banners and audio sounds.  going into the Firefox settings of the notification settings. it has notifications on, banner notifications on, notification center notifications enabled, sounds enabled, and the priority is set to normal. the only other option is to hide notifications on the lock screen. which is disabled. but i'm still not getting the notifications, is there anything else i need to change? 

as mentioned above i can change: 'alerts.useSystemBackend' to false, i haven't done that yet since that seems to be the direction FF is going and don't know if that will cause issues later on if it'd disabled with not getting other possible notifications.
